docs/docs/changelog.md

```
@@ -4,11 +4,63 @@
 <!-- https://github.com/fossunited/fossunited/pull/ -->
 <!-- https://github.com/fossunited/fossunited/issues -->
 
-## FossUnited
-
 📌 **Project Board**: [FOSS United GitHub Project](https://github.com/orgs/fossunited/projects/3/views/1)
 
-### August 2025
+## September 2025
+| Metric        | Count |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)